config.default.php

```diff
@@ -125,6 +125,9 @@
 		#   0 for an unlimited number of accounts
 		#   1 is to not allow user registrations (1 is corresponding to the admin account)
 		'max_registrations' => 1,
+
+		# Max amount of bytes that are allowed for upload of custom favicon
+		'max_favicon_upload_size' => 1 * 1024 * 1024, # 1 MB
```

Comment:
```suggestion
		'max_favicon_upload_size' => 1048576,	# 1 MiB
```

Good enough with the comment to avoid some multiplications in a file that is used in every single request